app/src/routes.js

```diff
@@ -207,23 +207,33 @@ export default (store) => {
     { path: `/404`, component: () => import(`./components/common/Page404`) },
     {
       path: `/privacy`,
-      component: () => import(`./components/common/PagePrivacy`),
+      beforeEnter() {
+        location.href = `https://lunie.io/privacy`
+      },
     },
     {
       path: `/terms`,
-      component: () => import(`./components/common/PageTerms`),
+      beforeEnter() {
+        location.href = `https://lunie.io/terms`
+      },
     },
     {
       path: `/security`,
-      component: () => import(`./components/common/PageSecurity`),
+      beforeEnter() {
+        location.href = `https://lunie.io/security`
+      },
     },
     {
       path: `/about`,
-      component: () => import(`./components/common/PageAbout`),
+      beforeEnter() {
+        location.href = `https://lunie.io/about`
+      },
     },
     {
       path: `/careers`,
-      component: () => import(`./components/common/PageCareers`),
+      beforeEnter() {
+        location.href = `https://lunie.io/careers`
```

Comment:
```suggestion
        location.href = `https://angel.co/company/lunie`
```

Reply:
this ok?

Reply:
why don't we have the careers page in https://lunie.io as well?

Reply:
I think it is ok to just use Angelist. I wouldn't mind as a developer looking for a job